data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/berserk/berserk_affect_everybody.cfg

```diff
@@ -0,0 +1,47 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [berserk]affect_allies=
+##
+# Actions:
+# Give the side 1 leader a berserk ability with affect_allies, affect_enemies, and affect_self
+# Make side 1 and side 2 allies
+# Have the side 1 leader attack the side 3 leader with both its weapons
+# Have the side 2 leader attack the side 4 leader with both its weapons
+# Have the side 4 leader attack the side 1 leader with both its weapons
+##
+# Expected end state:
+# The side 1 leader's first weapon strikes 10 times (2*5)
+# The side 1 leader's second weapon strikes 20 times (4*5)
+# The side 2 leader's first weapon strikes 10 times (2*5)
+# The side 2 leader's second weapon strikes 20 times (4*5)
+# The side 4 leader's first weapon strikes 10 times (2*5)
+# The side 4 leader's second weapon strikes 20 times (4*5)
+#####
+{COMMON_KEEP_A_B_C_D_UNIT_TEST "berserk_affect_everybody" (
+    [event]
+        name = start
+
+        [modify_side]
+            side=1,2
+            team_name=allies
+        [/modify_side]
+
+        {SET_HP VALUE=100}
```

Comment:
This seems around the range that the tests could intermittently fail when alice kills charlie or bob kills dave.

Reply:
I'm not sure how this could intermittently fail?